<!DOCTYPE html>
<html>
    <head>
        <title>
            Achievements
        </title><!-- for-mobile-apps -->
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <meta name="keywords" content="">
        <script type="application/x-javascript">
        addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false);
                function hideURLbar(){ window.scrollTo(0,1); } 
        </script><!-- //for-mobile-apps -->
        <link href="css/bootstrap.css" rel="stylesheet" type="text/css" media="all">
        <link href="css/style.css" rel="stylesheet" type="text/css" media="all"><!-- js -->

        <script src="js/jquery-1.11.1.min.js"></script><!-- //js -->
        <!-- fonts -->
        <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
        <link href='https://fonts.googleapis.com/css?family=Days+One' rel='stylesheet' type='text/css'><!-- //fonts -->
        <!-- start-smoth-scrolling -->

        <script type="text/javascript" src="js/move-top.js"></script>
        <script type="text/javascript" src="js/easing.js"></script>
        <script type="text/javascript">
        jQuery(document).ready(function ($) {
            $(".scroll").click(function (event) {
                event.preventDefault();
                $('html,body').animate({ scrollTop: $(this.hash).offset().top }, 1000);
            });
        });
        </script><!-- start-smoth-scrolling -->
    </head>
    <body>
        <!-- banner -->
        <div class="header">
            <div class="container">
                <div class="header-nav">
                    <nav class="navbar navbar-default">
                        <!-- Brand and toggle get grouped for better mobile display -->
                        <div class="navbar-header">
                            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span></button>
                            <h1>
                                <a class="navbar-brand" href="index.html">电磁频谱智能通信实验室
                                <p>
                                    ESCI Lab
                                </p></a>
                            </h1>
                        </div><!-- Collect the nav links, forms, and other content for toggling -->
                        <div class="collapse navbar-collapse nav-wil" id="bs-example-navbar-collapse-1">
                            <ul class="nav navbar-nav">
                                <li>
                                    <a href="index.html">
                                    <h4>
                                        首页
                                    </h4>
                                    <h5>
                                        Home
                                    </h5></a>
                                </li>
                                <li>
                                    <a href="area.html">
                                    <h4>
                                        研究方向
                                    </h4>
                                    <h5>
                                        Research
                                    </h5></a>
                                </li>
                                <li>
                                    <a href="people.html">
                                    <h4>
                                        研究团队
                                    </h4>
                                    <h5>
                                        People
                                    </h5></a>
                                </li>
                                <li>
                                    <a class="active" href="achievements.html">
                                    <h4>
                                        科研成果
                                    </h4>
                                    <h5>
                                        Achievements
                                    </h5></a>
                                </li>
                                <li>
                                    <a href="platform.html">
                                    <h4>
                                        实验平台
                                    </h4>
                                    <h5>
                                        Platform
                                    </h5></a>
                                </li>
                                <li>
                                    <a href="contact.html">
                                    <h4>
                                        联系我们
                                    </h4>
                                    <h5>
                                        Contact
                                    </h5></a>
                                </li>
                                <li>
                                    <a href="#">
                                    <h4>
                                        内部管理
                                    </h4>
                                    <h5>
                                        （预留）
                                    </h5></a>
                                </li>
                            </ul>
                        </div><!-- /navbar-collapse -->
                        <!-- search-scripts -->
                        <script src="js/classie.js"></script> 
                        <script src="js/uisearch.js"></script> 
                        <script>

                            new UISearch(document.getElementById('sb-search'));
                        </script> <!-- //search-scripts -->
                    </nav>
                </div>
            </div>
        </div>
        <div class="banner-bottom">
            <h3 style="color: #0095b6">
                科研成果
            </h3>
            <h3 style="font-size: 24px; color: #999; margin-top: 10px;">
                Achievements
            </h3>
        </div>
        <div class="typrography" style="padding-top: 0px;">
            <div class="container">
                <div class="bs-docs-example">
                    <table class="table table-hover table-bordered">
                        <thead>
                            <tr>
                                <th style="width: 40px">
                                    Index
                                </th>
                                <th>
                                    Publication Since Joining Hubei University
                                </th>
                                <th style="width: 120px">
                                    Date
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>
                                    1
                                </td>
                                <td>
                                    <a href="detail.html" target="_blank">Yang S*#, Vera J. M*, Savvakis G., Moskvin O. V., Yang Y., McIlwain S. J., Lyu Y., Zinonos I., Hebert A. S., Coon J. J., Bates D. M., Sato T. K., Brown S. D., Himmel M. E., Zhang M., Landick R., Pappas K. M#, Zhang Y*#. 2018. Complete genome sequence and the expression pattern of plasmids of the model ethanologen Zymomonas mobilis ZM4 and its xylose-utilizing derivatives 8b and 2032. Biotechnology for Biofuels. 11：125</a>
                                </td>
                                <td>
                                    2018
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    2
                                </td>
                                <td>
                                    <a href="detail.html" target="_blank">2.Wang X*., He Q. *, Yang Y. *, Wang J., Haning K., Hu Y., Wu B., He M., Zhang Y., Bao J., Contreras L. M. # and Yang S#. 2018. Advances and prospects in metabolic engineering of Zymomonas mobilis. Metabolic Engineering.</a>
                                </td>
                                <td>
                                    2018
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    3
                                </td>
                                <td>
                                    Yang S#, M. A. Franden, Q. Yang, Y.-C. Chou, M. Zhang and P. T. Pienkos#. 2018. Identification of inhibitors in lignocellulosic slurries and determination of their effect on hydrocarbon-producing microorganisms. Frontiers in Bioengineering and Biotechnology 6:23. doi: 10.3389/fbioe.2018.00023.
                                </td>
                                <td>
                                    2018
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    4
                                </td>
                                <td>
                                    Yang Y, Hu M, Tang Y, Geng B, Qiu M, He Q, Chen S, Wang X#, Yang S#. 2018. Progress and perspective on lignocellulosic hydrolysate inhibitor tolerance improvement in Zymomonas mobilis. Bioresources and Bioprocessing, 5(1):6.
                                </td>
                                <td>
                                    2018
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    5
                                </td>
                                <td>
                                    Cai, D., S. Hu, Y. Chen, L. Liu, S. Yang, X. Ma and S. Chen. 2018. Enhanced Production of Poly-γ-glutamic acid by Overexpression of the Global Anaerobic Regulator Fnr in Bacillus licheniformis WX-02. Applied Biochemistry and Biotechnology <a href="https://doi.org/10.1007/s12010-018-2693-7" target="_blank">https://doi.org/10.1007/s12010-018-2693-7</a>
                                </td>
                                <td>
                                    2018
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    6
                                </td>
                                <td>
                                    Huo, Y., Y. Zhan, Q. Wang, S. Li, Yang, S., C. T. Nomura, C. Wang and S. Chen. 2017. Acetolactate synthase (AlsS) in Bacillus licheniformis WX-02: enzymatic properties and efficient functions for acetoin/butanediol and L-valine biosynthesis. Bioprocess Biosyst Eng 41(1): 87-96.
                                </td>
                                <td>
                                    2017
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    7
                                </td>
                                <td>
                                    Xu, Q., E. P. Knoshaug, W. Wang, M. Alahuhta, J. O. Baker, Yang, S., T. Vander Wall, S. R. Decker, M. E. Himmel, M. Zhang, and H. Wei. 2017. Expression and secretion of fungal endoglucanase II and chimeric cellobiohydrolase I in the oleaginous yeast Lipomyces starkeyi. Microb Cell Fact 16(1): 126.
                                </td>
                                <td>
                                    2017
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    8
                                </td>
                                <td>
                                    Cho, S. H., K. Haning, W. Shen, C. Blome, R. Li, Yang S., and L. M. Contreras. 2017. Identification and characterization of 5′untranslated regions (5′UTRs) in Zymomonas mobilis as regulatory biological parts. Frontiers in Microbiology 8:2432.
                                </td>
                                <td>
                                    2017
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    9
                                </td>
                                <td>
                                    Yang, S. *#, Fei., Q.*, Zhang, Y., Contreras, L. M., Utturkar, S. M., Brown, S. D., Himmel, M.E., and Zhang, M#. 2016. Zymomonas mobilis as a model system for production of biofuels and biochemicals. Microbial Biotechnology. 9(6), 699–717.
                                </td>
                                <td>
                                    2016
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    10
                                </td>
                                <td>
                                    Yang, S. #, Mohagheghi, A., Chou, Y-C., Franden, M. A., Chen, X. W., Dowe, N., Himmel, M., and Zhang, M. # 2016. Metabolic engineering of Zymomonas mobilis for production of 2,3-butanediol from lignocellulosic biomass sugars. Biotechnology for Biofuels 9:189.
                                </td>
                                <td>
                                    2016
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    11
                                </td>
                                <td>
                                    Xu, Q., Resch, M., Podkaminer, K., Yang, S., Baker, J., Donohoe, B., Wilson, C., Klingeman, D. M., Olson, D., Decker, S., Giannone, R. J., Hettich, R. L., Brown, S. D., Lynd, L. R., Bayer, E. A., Himmel, M. E., and Bomble, Y. 2016. Dramatic performance of Clostridium thermocellum explained by its wide range of cellulase modalities. Science Advances. 2(2): e1501254.
                                </td>
                                <td>
                                    2016
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    12
                                </td>
                                <td>
                                    Yang, S. #, Wang, W. #, Wei, H., Wychen, S. V., Pienkos, P. T., Zhang, M., and Himmel, M. 2016. Impact of nitrogen deficiency strategies on lipid production for yeast and fungal species. Energies. 9:685.
                                </td>
                                <td>
                                    2016
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    13
                                </td>
                                <td>
                                    Shu, M., Shen, W., Yang, S., Wang, X., Wang, F., Wang, Y., and Ma, L. 2016. High-level expression and characterization of a novel serine protease in Pichia pastoris by multi-copy integration. Enzyme Microb Technol. 92:56-66.
                                </td>
                                <td>
                                    2016
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    <table class="table table-hover table-bordered">
                        <thead>
                            <tr>
                                <th style="width: 40px">
                                    Index
                                </th>
                                <th>
                                    Previous Publication
                                </th>
                                <th style="width: 120px">
                                    Date
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td colspan="3">
                                    <b>Biofuels：</b>
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    14
                                </td>
                                <td>
                                    Yang, S.#, Franden, A., Chou, Y-C., Brown, S. D., Pienkos, P. T., and Zhang, M#. 2014. Insights into acetate toxicity in Zymomonas mobilis 8b using different substrates. Biotechnology for Biofuels 7:140.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    15
                                </td>
                                <td>
                                    Yang, S.#, Pan, C., Hurst, G. B., Dice, L., Davison, B. H., and Brown, S. D.#. 2014. Elucidation of Zymomonas mobilis physiology and stress responses to acetate by quantitative proteomics and transcriptomics. Front. Microbiol. 5:246.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    16
                                </td>
                                <td>
                                    Yang, S., Pan, C., Tschaplinski, T. J., Hurst, G.B. Engle, N. L., Zhou, W., Dam, P., Xu, Y., Rodriguez, M. Jr., Dice, L., Johnson, C. M., Davison, B. H., and Brown, S. D. 2013. Systems biology analysis of Zymomonas mobilis ZM4 ethanol stress responses. PLoS ONE 8(7): e68886.
                                </td>
                                <td>
                                    2013
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    17
                                </td>
                                <td>
                                    Yang, S., Land, M. L., Klingeman, D. M., Pelletier, D. A., Lu, S. T., Martin, S. L., Guo, H. B., Smith, J. C., and Brown, S. D. 2010. Paradigm for industrial strain improvement identifies sodium acetate tolerance loci in Zymomonas mobilis and Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA. 107: 10395–10400.
                                </td>
                                <td>
                                    2010
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    18
                                </td>
                                <td>
                                    Yang, S., Pelletier, D. A., Lu, S. T, and Brown, S. D. 2010. The Zymomonas mobilis regulator Hfq and related Saccharomyces cerevisiae proteins contribute to tolerance against multiple lignocellulosic pretreatment inhibitors. BMC Microbiology 10: 135.
                                </td>
                                <td>
                                    2010
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    19
                                </td>
                                <td>
                                    Yang, S., Pappas, K. M., Hauser, L. J., Land, M. L., Chen, G-L, Hurst, G.B. et al. 2009. Improved genome annotation for Zymomonas mobilis. Nature Biotechnology 27: 893 –4.
                                </td>
                                <td>
                                    2009
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    20
                                </td>
                                <td>
                                    Yang, S., Tschaplinski, T. J., Engle, N. L., Carroll, S. L., Martin, S. L., Davison, B. H., Palumbo, A. V., Rodriguez, M. Jr., and Brown, D., S. 2009. Transcriptomic and metabolomic profiling of Zymomonas mobilis during aerobic and anaerobic fermentation. BMC Genomics 10: 34. (Highly accessed).
                                </td>
                                <td>
                                    2009
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    21
                                </td>
                                <td>
                                    Wang, W*#., Yang, S.*#, Pienkos, P. T., and Johnson, D. 2014. Connecting lignin-degradation pathway with pretreatment inhibitor sensitivity of Cupriavidus necator. Front. Microbiol. 5:247.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    22
                                </td>
                                <td>
                                    Yang, S.#, Guarnieri, M. T., Smolinski, S., Ghirardi, M., and Pienkos, P. T. 2013. De novo transcriptomic analysis of hydrogen production in the green alga Chlamydomonas moewusii through RNA-Seq. Biotechnology for Biofuels 6:118.
                                </td>
                                <td>
                                    2013
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    23
                                </td>
                                <td>
                                    Wilson, C. M.*, Yang, S.*, Rodriguez, M. Jr., Ma, Q. , Johnson, C. M., Dice, L., Xu, Y., and Brown, S. D. 2013. Clostridium thermocellum transcriptomic profiles after exposure to furfural or heat stress. Biotechnology for Biofuels 6:131.
                                </td>
                                <td>
                                    2013
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    24
                                </td>
                                <td>
                                    Yang, S., Giannone, R. J., Dice, L., Yang, Z. K., Engle, N. L., Tschaplinski, T. J., Hettich, R. L., and Brown, S. D. 2012. Clostridium thermocellum ATCC27405 transcriptomic, metabolomic and proteomic profiles after ethanol stress. BMC Genomics 13:336.
                                </td>
                                <td>
                                    2012
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    25
                                </td>
                                <td>
                                    Chou, Y-C., Linger, J., Yang, S., and Zhang, M. 2015. Genetic engineering and improvement of a Zymomonas mobilis for arabinose utilization and its performance on pretreated corn stover hydrolysate. J. Biotechnol. Biomater. 5:2.
                                </td>
                                <td>
                                    2015
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    26
                                </td>
                                <td>
                                    Mohagheghi, A., Linger, J., Yang, S., Smith, H., Dowe, N., Zhang, M., and Pienkos, P. T. 2015. Improving a recombinant Zymomonas mobilis strain 8b through continuous adaptation on dilute acid pretreated corn stover hydrolysate. Biotechnology for Biofuels 8:55.
                                </td>
                                <td>
                                    2015
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    27
                                </td>
                                <td>
                                    Zeng, Y., Zhao, S., Yang, S., and Ding, S. 2014. Lignin plays the negative role in the biochemical process for producing lignocellulosic biofuels. Curr. Opin. Biotechnol. 27: 38-45.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    28
                                </td>
                                <td>
                                    Mohagheghi, A., Linger, J., Smith, H., Yang, S., Dowe, N., and Pienkos, P. T. 2014. Improving xylose utilization by recombinant Zymomonas mobilis strain 8b through adaptation using 2-deoxyglucose. Biotechnology for Biofuels 7: 19.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    29
                                </td>
                                <td>
                                    Wei, H., Fu, Y., Magnusson, L., Baker, JO., Maness, PC., Xu, Q., Yang, S., Bowersox, A., Bogorad, I., Wang, W., Tucker, MP., Himmel, ME., Ding, S. 2014. Comparison of transcriptional profiles of Clostridium thermocellum grown on cellobiose and pretreated yellow poplar using RNA-Seq. Front. Microbiol. 5:142.
                                </td>
                                <td>
                                    2014
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    30
                                </td>
                                <td>
                                    Guarnieri, M. T., Nag, A., Yang, S., and Pienkos, P. T. 2013. Proteomic analysis of Chlorella vulgaris: Potential targets for enhanced lipid accumulation. J Proteomics. pii: S1874-3919(13)00278-9.
                                </td>
                                <td>
                                    2013
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    31
                                </td>
                                <td>
                                    Brown, S. D., Guss, A. M., Karpinets, T. V., Parks, J. M., Smolin, N., Yang, S., Land, M. L., et al. 2011. Mutant alcohol dehydrogenase leads to improved ethanol tolerance in Clostridium thermocellum. Proc. Natl. Acad. Sci. USA. 108: 13752-7.
                                </td>
                                <td>
                                    2011
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    32
                                </td>
                                <td>
                                    Bochner, B., Gomez, V., Ziman, M., Montgomery, S., Yang, S., and Brown, S. D. 2010. Phenotype MicroArray profiling of Zymomonas mobilis ZM4. Appl. Biochem. Biotech. 161:116-123.
                                </td>
                                <td>
                                    2010
                                </td>
                            </tr>
                            <tr>
                                <td colspan="3">
                                    <b>Molecular Microbe-Plant Interactions and others：</b>
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    33
                                </td>
                                <td>
                                    Yang, S., Peng, Q., Zhang, Q., Zou, L., Li, Y., Robert, C. et al. 2010. Genome-wide identification of HrpL-regulated genes in necrotrophic phytopathogen Dickeya dadantii 3937. PLoS ONE 5(10): e13472.
                                </td>
                                <td>
                                    2010
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    34
                                </td>
                                <td>
                                    Yang, S.*, Peng, Q.*, San Francisco, M., Wang, Y., Zeng, Q., and Yang, C.-H. 2008. Type III secretion system genes of Dickeya dadantii 3937 are induced by plant phenolic acids. PLoS ONE 3(8): e2973.
                                </td>
                                <td>
                                    2008
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    35
                                </td>
                                <td>
                                    Yang, S., Zhang, Q., Peng, Q., Yi, X., Chang, J. C., Reedy, R. M., Charkowski, A. O., and Yang, C.-H. 2008. Dynamic regulation of GacA in type III secretion system, pectinase gene expression, pellicle formation, and pathogenicity of Dickeya dadantii. Mol. Plant-Microbe Interact. 21: 133-142.
                                </td>
                                <td>
                                    2008
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    36
                                </td>
                                <td>
                                    Yang, S., Zhang, Q., Guo, J., Charkowski, A. O., Glick, B. R., Ibekwe, A. M., Cooksey, D. A., and Yang, C.-H. 2007. Global effect of indole-3-acetic acid (IAA) biosynthesis on multiple virulence factors of Erwinia chrysanthemi 3937. Appl. Env. Microbiol. 73:1079-1088.
                                </td>
                                <td>
                                    2007
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    37
                                </td>
                                <td>
                                    Peng, Q.*, Yang, S.*, Charkowski, A. O., Yap, M. N., Steeber, D. A., Keen, N. T., and Yang, C.-H. 2006. Population behavior analysis of dspE and pelD expression in Erwinia chrysanthemi 3937. Mol. Plant-Microbe Interact. 19: 451-7.
                                </td>
                                <td>
                                    2006
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    38
                                </td>
                                <td>
                                    Yang, S., Perna, N. T., Cooksey, D. A., Okinaka, Y., Lindow, S. E., Ibekwe, A. M., Keen, N. T., and Yang, C-H. 2004. Genome-wide identification of plant-upregulated genes of Erwinia chrysanthemi 3937 using a GFP-based IVET leaf array. Mol. Plant-Microbe Interact. 17: 999-1008.
                                </td>
                                <td>
                                    2004
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    39
                                </td>
                                <td>
                                    Chou, W-C., Ma, Q., Yang, S., Cao, S., Brown, S. D., and Xu, Y. 2015. Analysis of strand-specific RNA-seq data using machine learning reveals the structures of transcription units in Clostridium thermocellum. Nucleic Acids Res. doi: 10.1093/nar/gkv177.
                                </td>
                                <td>
                                    2015
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    40
                                </td>
                                <td>
                                    Wei, H., Brunecky, R., Donohoe, BS., Ding, S., Ciesielski, PC., Yang, S., Tucker, MP., and Himmel, ME. 2015. Identifying the relatively abundant, ionically bound cell wall and intracellular glycoside hydrolases and carbohydrate esterases in late growth stage Arabidopsis stems: implications for the genetic engineering of bioenergy crops. Front. Plant Science. 6:315.
                                </td>
                                <td>
                                    2015
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    41
                                </td>
                                <td>
                                    Glasner, J. D., Yang, C.-H, Reverchon, S., Hugouvieux-Cotte-Pattat, N., Condemine, G., Bohin, J., Van Gijsegem, F., Yang, S., etc. 2011. Genome sequence of the plant pathogenic bacterium Dickeya dadantii 3937. J. Bacteriol. 193:2076-7.
                                </td>
                                <td>
                                    2011
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    42
                                </td>
                                <td>
                                    Okinaka, Y. Perna, N. T., Yang, S. Keen, N. T., and Yang, C-H. 2006. Identification of potential virulence genes in Erwinia chrysanthemi 3937; transposon insertion into plant up-regulated genes. J. Gen. Plant Pathol. 72:360-8.
                                </td>
                                <td>
                                    2006
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    43
                                </td>
                                <td>
                                    Schijven, J. F., Bradford, S. A., Yang, S. 2004. Release of Cryptosporidium and Giardia from dairy cattle manure: physical factors. J. Environ. Qual. 33:1499-1508.
                                </td>
                                <td>
                                    2004
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    44
                                </td>
                                <td>
                                    Yang, S., Fang, C., and Zhang, L. 2000. A staining method used for observing protoplast under the light microscope. Microbiology (Chinese) 27:55-7.
                                </td>
                                <td>
                                    2000
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    <table class="table table-hover table-bordered">
                        <thead>
                            <tr>
                                <th style="width: 40px">
                                    Index
                                </th>
                                <th>
                                    Patents
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>
                                    1
                                </td>
                                <td>
                                    Yang, S., Linger, J., Franden, A., Pienkos, P. T., and Zhang, M. US14265039. Biocatalysts with enhanced inhibitor tolerance.
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    2
                                </td>
                                <td>
                                    Brown, S., and Yang, S. US20130078691. Microorganisms having enhanced resistance to acetate and related compositions and methods of Use.
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    3
                                </td>
                                <td>
                                    Brown, S., Guss, A., Yang, S., Karpinets, T., and Lynd, L. US20110287499. Nucleic acid molecules conferring enhanced ethanol tolerance and microorganisms having enhanced tolerance to ethanol.
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    4
                                </td>
                                <td>
                                    Brown, S., and Yang, S. US20100311137. Microorganisms having enhanced tolerance to inhibitors and stress.
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    5
                                </td>
                                <td>
                                    Yang, C.-H., and Yang, S. US20100249234 and WIPO WO/2008/124836. Methods of reducing virulence in bacteria.
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    <table class="table table-hover table-bordered">
                        <thead>
                            <tr>
                                <th style="width: 40px">
                                    Index
                                </th>
                                <th>
                                    Book Chapters
                                </th>
                                <th style="width: 120px">
                                    Date
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>
                                    1
                                </td>
                                <td>
                                    Yang, S., Wang, W., Wei, H., Himmel, M., and Zhang, M. 2015. Identification of genetic targets to improve lignocellulosic hydrocarbon production in Trichoderma reesei using public genomic and transcriptomic datasets. P177-196. In Direct Microbial Conversion of Biomass to Advanced Biofuels. Edited by Michael E Himmel. Elsevier. New York, US.
                                </td>
                                <td>
                                    2015
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    2
                                </td>
                                <td>
                                    Yang, S., Klingeman, D. M., and Brown, S. D. 2012. Ethanol-tolerant gene identification in Clostridium thermocellum using pyro-resequencing for metabolic engineering. P111-136. In Methods in Molecular Biology: Microbial Metabolic Engineering. Edited by Cheng Q. Human Press. New York, US.
                                </td>
                                <td>
                                    2012
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    3
                                </td>
                                <td>
                                    Yang, S., Keller, M., and Brown, S. D. 2011. Genomics on pretreatment inhibitor tolerance of Zymomonas mobilis. P161-176. In: Microbial stress tolerance: from genomics to biofuels Microbiology monographs (Springer series). Edited by Liu Z. L. Springer. Heidelberg, Germany.
                                </td>
                                <td>
                                    2011
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    4
                                </td>
                                <td>
                                    Yang, C.-H., and Yang, S. 2008. Managing bacterial plant diseases by modulating quorum sensing and Type III secretory systems. P16-57. In: Biotechnology and Plant Disease Management. Eds. Punja Z.K., De Boer S.H., and Sanfacon H. CABI Publishing. Oxfordshire. UK.
                                </td>
                                <td>
                                    2008
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    <table class="table table-hover table-bordered">
                        <thead>
                            <tr>
                                <th style="width: 40px">
                                    Index
                                </th>
                                <th>
                                    Presentations: (&gt;60 total, 45 first-authored in ASM, APS, SIMB, and ICMS etc.)
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td>
                                    1
                                </td>
                                <td>
                                    Invited talks on next-generation sequencing and systems biology for biofuel biocatalyst development at international conference of: ACS Annual Meeting; Society for Industrial Microbiology & Biotechnology (SIMB) Annual Meeting; Symposium on Biotechnology for Fuels and Chemicals; as well as DuPont, IneosBio, Wuhan University, and Beijing Institute of Genomics etc.
                                </td>
                            </tr>
                        </tbody>
                    </table>
                </div>
            </div>
        </div>
        <div class="footer-top" style="clear: both;">
            <p>
                Copyright © 2023.南京航空航天大学周福辉教授版权所有 All rights reserved.
            </p>
        </div><!-- for bootstrap working -->
        <script src="js/bootstrap.js"></script> <!-- //for bootstrap working -->
         <!-- smooth scrolling -->
         
        <script type="text/javascript">

        $(document).ready(function () {
            /*
            var defaults = {
            containerID: 'toTop', // fading element id
            containerHoverID: 'toTopHover', // fading element hover id
            scrollSpeed: 1200,
            easingType: 'linear' 
            };
            */
            $().UItoTop({ easingType: 'easeOutQuart' });
        });
        </script> <a href="#" id="toTop" style="display: block;"><span id="toTopHover" style="opacity: 1;"></span></a> <!-- //smooth scrolling -->
    </body>
</html>